### PR TITLE
Integrating Kved and Ruo and Rfop

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: ratu/migrations/0037_auto_20200423_1237.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    dependencies = [
+        ('ratu', '0036_auto_20200422_1225'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='kzed',
+            name='name',
+            field=models.CharField(max_length=1),
```
Comment (criticus) [inline ratu/migrations/0037_auto_20200423_1237.py:17]: is this correct? one symbol?

Comment (IegorSoboliev) [reply ratu/migrations/0037_auto_20200423_1237.py:17]: no, sorry. Have changed that in the next commits

File: ratu/services/kved_service.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    #list of models for clearing DB
+    tables=[
+        Section,
+        Division,
+        Group, 
+        Kved
+    ]
+
+    def save_default_kved(self):
```
Comment (criticus) [inline ratu/services/kved_service.py:20]: please add comment to delete this method once process() is fixed

File: ratu/services/rfop_service.py [hunk 3]
```diff
@@ -64,22 +65,20 @@ def save_to_state_table(self, record):
...
-    #writing entry to kved table       
-    def save_to_kved_table(self, record):
-        if record['KVED']:
-            kved_name=record['KVED']
+            
+    #verifying kved 
+    def get_kved_from_DB(self, record):
+        if record['KVED'] and record['KVED'] != None and record['KVED'] != "None":
+            #in xml record we have code and name of the kved together in one string. Here we are getting only code
+            kved_code = record['KVED'].split(" ")[0]
```
Comment (criticus) [inline ratu/services/rfop_service.py:73]: @Anntroy added a function to get the first word. Can you use it here too?

File: ratu/services/rfop_service.py [hunk 4]
```diff
@@ -94,4 +93,5 @@ def save_to_rfop_table(self, record, state, kved):
     print(
         'Rfop_class already imported. For start rewriting RFOP to the DB run > RfopConverter().process()\n',
         'For clear RFOP tables run > RfopConverter().clear_db()'
-        )
\ No newline at end of file
+        )
+
```
Comment (criticus) [inline ratu/services/rfop_service.py:97]: extra line

File: ratu/services/ruo_service.py [hunk 5]
```diff
@@ -77,35 +69,22 @@ def save_to_state_table(self, record):
...
-    #writing entry to kzed table       
-    def get_kzed(self, record):
+        
+    #verifying kved 
+    def get_kved_from_DB(self, record):
         if record['KVED']:
-            record_as_list = record['KVED'].split(" ")
-            kved_code=record_as_list[0]
+            #in xml record we have code and name of the kved together in one string. Here we are getting only code
+            kved_code = record['KVED'].split(" ")[0]
```
Comment (criticus) [inline ratu/services/ruo_service.py:77]: same here, use @Anntroy's method

File: ratu/services/kved_service.py (hunk 2 above)
Comment (IegorSoboliev) [reply ratu/services/kved_service.py:21]: Done

File: ratu/services/rfop_service.py (hunk 3 above)
Comment (IegorSoboliev) [reply ratu/services/rfop_service.py:73]: Sure

File: ratu/services/rfop_service.py (hunk 4 above)
Comment (IegorSoboliev) [reply ratu/services/rfop_service.py]: Done

File: ratu/services/ruo_service.py (hunk 5 above)
Comment (IegorSoboliev) [reply ratu/services/ruo_service.py:77]: Sure

**Pull request merged**